src/platform/backends/lxd/lxd_vm_image_vault.cpp

```diff
@@ -226,11 +228,13 @@ bool mp::LXDVMImageVault::has_record_for(const std::string& name)
 
 void mp::LXDVMImageVault::prune_expired_images()
 {
+    mpl::log(mpl::Level::trace, category, "Pruning expired images not implemented");
 }
 
 void mp::LXDVMImageVault::update_images(const FetchType& fetch_type, const PrepareAction& prepare,
                                         const ProgressMonitor& monitor)
 {
+    mpl::log(mpl::Level::trace, category, "Updating images not implemented");
```

Comment:
I think I'd put these at `debug`, but :shrug:.

Reply:
#1594 implements this, so I think messing with this in this PR is unnecessary.